testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/markup.tests.js

```diff
@@ -7,6 +7,9 @@ const HTML_EDITOR_CLASS = 'dx-htmleditor';
 const QUILL_CONTAINER_CLASS = 'dx-quill-container';
 const HTML_EDITOR_CONTENT_CLASS = 'dx-htmleditor-content';
 const HTML_EDITOR_SUBMIT_ELEMENT_CLASS = 'dx-htmleditor-submit-element';
+const HTML_EDITOR_STYLING_MODE_OUTLINED_CLASS = 'dx-htmleditor-outlined';
```

Comment:
to my mind `HTML_EDITOR_OUTLINED_CLASS` (without `STYLING_MODE` in the middle) is more laconic and easy-to-read